pandapipes/std_types/std_types.py

```diff
@@ -5,7 +5,7 @@
 import os
 import warnings
 import re
-
+import numpy as np
```

Comment:
I think this import statement is not needed, would look better wrt codacy to remove.